packages/client/package.json

```diff
@@ -172,8 +173,6 @@
       "optional": true
     }
   },
-  "dependencies": {
-    "@prisma/engines-version": "5.6.0-8.582b416e6b0ed6f95140cdea85b3a49e2dc3e52b"
-  },
+  "dependencies": {},
```

Comment:
This change means that it will not be possible to change the engines version by overriding the dependency anymore.
Maybe not a blocker, but an important detail to note.